src/main/java/duke/storage/Storage.java

```diff
@@ -0,0 +1,104 @@
+package duke.storage;
+
+import java.nio.file.Files;
+import java.io.File;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Scanner;
```

Comment:
You may want to order your packages so that they are consistent, eg keep all from the same package together